app/Club.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Club extends Model
+{
+    protected $fillable = [
+        'name', 'email', 'creation_date','logo','description',
```

Comment:
Please instead of `creation_date`, put `created_at` and `updated_at`

Reply:
it means when the club is created not when it is added to database

Reply:
so we need we will have 3: `creation_date` plus the timestamps

Reply:
yes